src/app/app.component.ts

```diff
@@ -10,28 +13,63 @@ import { ContextDropdownModule } from './context-dropdown/context-dropdown.modul
     RouterOutlet,
     CommonModule,
     ContextDropdownModule,
+    MatMenuModule,
+    MatButtonModule,
   ],
   templateUrl: './app.component.html',
   styleUrl: './app.component.css',
 })
 export class AppComponent {
   title = 'context-dropdown';
+  imagePath = 'assets/placeholder.webp';
 
+  // reference to the MatMenuTrigger in the DOM
+  @ViewChild(MatMenuTrigger, { static: true }) matMenuTrigger!: MatMenuTrigger;
+  menuTopLeftPosition = { x: 0, y: 0 };
```

Comment:
move this around
rule:
variables
@inputs
@outouts
@viewchild
constructor
ngOnInit
getters
public functions
private functions
onDestroy